pkg/accesscontrol/policy_rule_index_test.go

```diff
@@ -236,3 +238,142 @@ func makeCRB(name string, roleRef rbacv1.RoleRef, subjects []rbacv1.Subject) *rb
 		Subjects: subjects,
 	}
 }
+
+func Test_addResourceAccess(t *testing.T) {
+	tests := []struct {
+		name      string
+		namespace string
+		rule      rbacv1.PolicyRule
+		want      AccessSet
+	}{
+		{
+			name:      "RoleBinding namespaces resource with empty names",
+			namespace: "test-ns",
+			rule: rbacv1.PolicyRule{
+				APIGroups:     []string{""},
```

Comment:
Please add test cases with multiple Resources and APIGroups.

Reply:
Added.